### ISSUE TITLE
potential API misuse in file agents\tf_agents\agents\sac\sac_agent.py:639

### ISSUE THREAD

**Issue by bugreportanon** [tensorflow/agents#869]
Hi, I just want to report a potential API misuse in your project. Please see the following details:

**Title**
potential API misuse in file agents\tf_agents\agents\sac\sac_agent.py:639

**File location**
agents\tf_agents\agents\sac\sac_agent.py:639

**Code snippet**

      # We do not update actor during alpha loss.
      unused_actions, log_pi = self._actions_and_log_probs(
          time_steps, training=False)
      entropy_diff = tf.stop_gradient(-log_pi - self._target_entropy)
      if self._use_log_alpha_in_alpha_loss:
        alpha_loss = (self._log_alpha * entropy_diff)
      else:

**Suggest fix**

Fixed:
```python
unused_actions, log_pi = self._actions_and_log_probs(
    time_steps, training=False)
entropy_diff = tf.stop_gradient(-log_pi - self._target_entropy)
if self._use_log_alpha_in_alpha_loss:
    alpha_loss = (self._log_alpha * tf.stop_gradient(entropy_diff))
else:
    # Add fix pattern 1
    alpha_loss = (self._log_alpha * entropy_diff)
```
